custom-metrics-stackdriver-adapter/adapter.go

```diff
@@ -144,12 +146,16 @@ func (sa *StackdriverAdapter) withCoreMetrics(translator *translator.Translator)
 		return err
 	}
 
-	pods := informers.Core().V1().Pods()
-	nodes := informers.Core().V1().Nodes()
-	if err := api.Install(provider, pods.Lister(), nodes.Lister(), server.GenericAPIServer); err != nil {
+	podInformer, nil := informers.ForResource(corev1.SchemeGroupVersion.WithResource("pods"))
+	if err != nil {
 		return err
 	}
 
+	nodes := informers.Core().V1().Nodes()
+	if err := api.Install(provider, podInformer.Lister(), nodes.Lister(), server.GenericAPIServer, []labels.Requirement{}); err != nil {
+		return nil
```

Comment:
you're returning nil error in a non-nil error check, likely a typo?

Reply:
Done.